diff --git a/lib/open_api_sdk/models/components/category.rb b/lib/open_api_sdk/models/components/category.rb                           
index 1d940f3..bbc0459 100644                                                                                                          
--- a/lib/open_api_sdk/models/components/category.rb                                                                                   
+++ b/lib/open_api_sdk/models/components/category.rb                                                                                   
@@ -16,7 +16,7 @@ module OpenApiSDK                                                                                                    
                                                                                                                                       
         field :id, Crystalline::Nilable.new(::Integer), { 'format_json': { 'letter_case': ::OpenApiSDK::Utils.field_name('id') } }    
                                                                                                                                       
-        field :name, Crystalline::Nilable.new(::String), { 'format_json': { 'letter_case': ::OpenApiSDK::Utils.field_name('name') } } 
+        field :name, Crystalline::Nilable.new(::Boolean), { 'format_json': { 'letter_case': ::OpenApiSDK::Utils.field_name('name') } }
                                                                                                                                       
                                                                                                                                       
         def initialize(id: nil, name: nil)                                                                                            
@@ -24,6 +24,9 @@ module OpenApiSDK                                                                                                    
           @name = name                                                                                                                
         end                                                                                                                           
                                                                                                                                       
+        def print                                                                                                                     
+          @id.to_s                                                                                                                    
+        end                                                                                                                           
                                                                                                                                       
         def ==(other)                                                                                                                 
           return false unless other.is_a? self.class                                                                                  
